### PR TITLE
PvD

### PR DESCRIPTION
**Pull request by danwing**
See how this feels.  Also as part of this change I brought DHCP and 3GPP PCO to be more equal to SVCB and PvD (but still left DHCP and 3GPP PCO thin for specification, as I think both are too straight forward to bother typing details at this point).

### CODE REVIEW

File: draft-wing-cidfi.md [hunk 1]
```diff
@@ -290,16 +345,33 @@ authorizes those network elements ({{client-authorizes}}).
...
+
+> Standardizing on all or some of these mechanisms is for further discussion.
+
+In all cases below,
+
+- if the discovery succeeds, the client follows the processing in
+  {{client-authorizes}}.
+
+- if the discovery failed (i.e., the client concludes that the local
+  network does not support CIDFI), the processing stops.
```
Comment (tireddy2) [inline draft-wing-cidfi.md:360]: It is possible that the local network may not have CIDFI-aware network elements but the ISP may have CIDFI-aware network elements.  Replace "the client concludes that the local network does not support CIDFI" with "the client concludes the local network and ISP do not support CIDIF".

File: draft-wing-cidfi.md [hunk 2]
```diff
@@ -249,7 +255,18 @@ client and server and with the participating CIDFI network elements.
...
-# Network Preparation: DNS SVCB Records
+# Network Preparation
+
+The network is configured to advertise its support for CIDFI.
+
+For this step, four mechanisms are described in this document: DNS
+SVCB records, IPv6 Provisioning Domains (PvD), DHCP, and 3GPP PCO.
+These are described below.
+
+> Standardizing on all or some of these mechanisms is for further discussion.
```
Comment (tireddy2) [inline draft-wing-cidfi.md:266]: I don't think standardizing 3GPP PCO is in the scope of IETF. Why do we have to discuss about "3GPP PCO" ?

Comment (boucadair) [reply draft-wing-cidfi.md:266]: Agree we don't need to specify the PCO IE in this draft as this is a 3GPP thing. 

That's said, I think this is covered by "or some of these" in the text proposed by Dan. I think the current wording is OK here.

**Review by boucadair — COMMENTED** [review]
Thanks for taking care of this. The mention of RFC7839 is sufficient at this stage for DHCP to avoid overloading the spec with discovery details.

**Pull request merged**